frontend/packages/topology/locales/zh/topology.json

```diff
@@ -0,0 +1,106 @@
+{
+  "Move connector": "移动连接器",
+  "Delete connector": "删除连接器",
+  "Delete Application": "删除应用程序",
+  "Edit Application grouping": "编辑应用程序组",
+  "Edit {{applicationName}}": "编辑 {{applicationName}}",
+  "Select an Application": "选择一个应用程序",
+  "Create Application": "创建应用程序",
+  "No Application group": "无应用程序组",
+  "Warning: the Application grouping already exists.": "警告：应用程序组已存在。",
+  "A unique name given to the Application grouping to label your resources.": "为应用程序组指定一个唯一名称以标记您的资源。",
+  "Application": "应用程序",
+  "Select an Application for your grouping or no Application group to not use an Application grouping.": "为您的组选择一个应用程序，或选择没有应用程序组以不使用应用程序分组。",
+  "Application name": "应用程序名称",
+  "all applications": "所有应用程序",
+  "no application group": "无应用程序组",
+  "Error moving connection": "移动连接出错",
+  "Error creating connection": "创建连接出错",
+  "Add resources": "添加资源",
+  "Build {{status}}": "构建 {{status}}",
+  "Monitoring alert": "监控警报",
+  "Create a visual connector": "创建可视连接器",
+  "Edit source code": "编辑源代码",
+  "Open URL": "打开 URI",
+  "Zoom In": "放大",
+  "Zoom Out": "缩小",
```

Comment:
`Zoom out` ?

Reply:
Opened https://bugzilla.redhat.com/show_bug.cgi?id=1908467.

Reply:
This is all set; the bug PR merged.